base_location/models/res_city.py

```diff
@@ -0,0 +1,16 @@
+# Copyright 2018 Aitor Bouzas <aitor.bouzas@adaptivecity.com>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo import fields, models
+
+
+class City(models.Model):
+    _inherit = 'res.city'
+
+    code = fields.Char(
```

Comment:
Do you think this code brings too much value? Are there official codes for the cities?